test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+
+# Starting up demo cluster
+source "/usr/local/greenplum-db-devel/greenplum_path.sh"
+pushd gpdb_src/gpAux/gpdemo
+make create-demo-cluster WITH_MIRRORS=true
+source gpdemo-env.sh
+popd
+
+# Creating backup and log directories for pgbackrest
+TEST_NAME=$(basename "${0%.sh}")
+mkdir -p "$PGBACKREST_TEST_DIR/logs/$TEST_NAME"
+mkdir -p "$PGBACKREST_TEST_DIR/$TEST_NAME"
+DATADIR="${MASTER_DATA_DIRECTORY%*/*/*}"
+MASTER=${DATADIR}/qddir/demoDataDir-1
+PRIMARY1=${DATADIR}/dbfast1/demoDataDir0
+PRIMARY2=${DATADIR}/dbfast2/demoDataDir1
+PRIMARY3=${DATADIR}/dbfast3/demoDataDir2
+MIRROR1=${DATADIR}/dbfast_mirror1/demoDataDir0
+MIRROR2=${DATADIR}/dbfast_mirror2/demoDataDir1
+MIRROR3=${DATADIR}/dbfast_mirror3/demoDataDir2
+
+# Filling the pgbackrest.conf configuration file
+cat <<EOF > /etc/pgbackrest.conf
+[seg-1]
+pg1-path=$MASTER
+pg1-port=6000
```

Comment:
`$PGPORT`

Reply:
I've initially done similar stuff but mr. andr-sokolov was against https://github.com/arenadata/pgbackrest/pull/30#discussion_r1603170871. I guess current varian is more obvious

Reply:
He said about values, which you write at the script, I propose to use values, which will be autogenerated. You should not write `PGPORT=6000`, it has already done at the source file: `gpdemo-env.sh`, which you have already included into this script.